signer/src/storage/model.rs

```diff
@@ -108,6 +108,35 @@ pub struct WithdrawSigner {
     pub created_at: time::PrimitiveDateTime,
 }
 
+/// A transaction on either Bitcoin or Stacks
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "testing", derive(fake::Dummy))]
+pub struct Transaction {
+    txid: Bytes,
```

Comment:
Yeah, as you can tell I'm frequently shadowing names intentionally. (What you're pointing to is a third-party crate though :sweat_smile:). For structs and normal types I like to keep type names simple, and these are just bytes so that's a pretty accurate name I'd say. If you see any problem with this, I'm okay with changing the name but here I think this reads pretty nicely.

Reply:
Na, it's fine as is. I was just briefly surprised that it wasn't the other `Bytes` type (even though the alias was added prior to this PR).